lib/rspec/core/formatters/c_formatter.rb

```diff
@@ -0,0 +1,18 @@
+RSpec::Support.require_rspec_core "formatters/base_formatter"
+
+module RSpec
+  module Core
+    module Formatters
+      # @private
+      class CFormatter < BaseFormatter
+        Formatters.register self, :example_failed, :dump_profile
+
+        def example_failed(failure)
+          output.puts "#{failure.example.location}:#{failure.example.description}"
```

Comment:
Theres a potential flaw here with shared examples, as they won't know they're exact location

Reply:
Yeah, the formatter will only display the file and line of the failure (the failure line in the shared example), without the backtrace that helps figure out what path let to this bad situation.

Wrapping the backtrace on a single line look like a terrible idea, but I guess we can manage to have the human name of the way through the previous layers as the documentation formatter displays?

These failing tests (a single bogus test in a shared example):

```
TLSChecker::CertificateCheckerFactory
  #certificate_checkers_for
    is expected to be falsey (FAILED - 1)
    with a SMTP hostname
      first item
        is expected to be falsey (FAILED - 2)
```

Currently displays:

```
./spec/certificate_checker_factory_spec.rb:4:is expected to be falsey
./spec/certificate_checker_factory_spec.rb:4:is expected to be falsey
```

And could display:

```
./spec/certificate_checker_factory_spec.rb:4:TLSChecker::CertificateCheckerFactory #certificate_checkers_for is expected to be falsey
./spec/certificate_checker_factory_spec.rb:4:TLSChecker::CertificateCheckerFactory #certificate_checkers_for with a SMTP hostname first item is expected to be falsey
```

Maybe a bit too long…  We can also format this like `<failure message> (in <location>)` to avoid having the actual failure message too far away in the line:

```
./spec/certificate_checker_factory_spec.rb:4:is expected to be falsey (in TLSChecker::CertificateCheckerFactory #certificate_checkers_for)
./spec/certificate_checker_factory_spec.rb:4:is expected to be falsey (in TLSChecker::CertificateCheckerFactory #certificate_checkers_for with a SMTP hostname first item)
```

What do you think?

Reply:
This problem is one of the reasons why we use example ids to reproduce exactly what spec to run, there is no "one line" answer we're happy with.

Reply:
So let's keep it simple and only output the error message and not the "path" that let to this error.